### ISSUE TITLE
Substraction of onehot_invert in Identifier.loss() 

### ISSUE THREAD

**Issue by danlapko** [pudae/kaggle-humpback#2]
First of all, thanks for the great job.
It is clear how you are turning logits of flipped images to zero in loss function, but it is not obvious why in this line we need to subtract onehot_invert. Could you explain please?
https://github.com/pudae/kaggle-humpback/blob/c975332a99bec9c2485fea17c831f52f9a77736f/tasks/identifier.py#L148
